src/basic/unit-def.h

```diff
@@ -138,6 +138,7 @@ typedef enum ServiceState {
         SERVICE_RELOAD,            /* Reloading via ExecReload= */
         SERVICE_RELOAD_SIGNAL,     /* Reloading via SIGHUP requested */
         SERVICE_RELOAD_NOTIFY,     /* Waiting for READY=1 after RELOADING=1 notify */
+        SERVICE_RELOAD_MOUNTING,   /* Refreshing extensions for a reload request */
```

Comment:
I wouldn't call this "mounting". Later I'll be adding cred reloading and it would be a lot nicer if the names are more specific, i.e. `_RELOAD_EXTENTIONS`.

Or actually, call it _REFRESH_EXTENSIONS. As discussed at one of the meetings Lennart wanted refreshing without reloading to work. Let's be open to that.